readme.md

```diff
@@ -1 +1,77 @@
 Hello!
+
+# li
```

Comment:
Pretty please no carriage returns in long lines :)

Reply:
You mean the lines below?  How come?

Reply:
Leaving for now, can fix later.